nbconvert/postprocessors/serve.py

```diff
@@ -84,7 +97,14 @@ def postprocess(self, input):
         print("Serving your slides at %s" % url)
         print("Use Control-C to stop this server")
         if self.open_in_browser:
-            webbrowser.open(url, new=2)
+            try: 
+                browser = webbrowser.get(self.browser or None)
+                threading.Thread(target=(browser.open(url, new=2))).start()
```

Comment:
This won't actually do it in a thread - you're calling `browser.open()` in the main thread, and passing its return value as the `target=` parameter.

You can either use a lambda, like we [do in notebook](https://github.com/jupyter/notebook/blob/5.0.0/notebook/notebookapp.py#L1398), or pass args and kwargs separately, like this:

```python
Thread(target=browser.open, args=(url,), kwargs={'new': 2})
```

Both ways are a bit ugly, unfortunately.